src/stacked_hourglass/predictor.py

```diff
@@ -79,11 +90,11 @@ def estimate_joints(self, images, flip=False):
         is_batched = _check_batched(images)
         raw_images = images if is_batched else images.unsqueeze(0)
         heatmaps = self.estimate_heatmaps(raw_images, flip=flip).cpu()
-        coords = final_preds_untransformed(heatmaps, (64, 64))
+        coords = final_preds_untransformed(heatmaps, self.output_shape)
         # Rescale coords to pixel space of specified images.
         for i, image in enumerate(raw_images):
-            coords[i, :, 0] *= image.shape[-1] / 64
-            coords[i, :, 1] *= image.shape[-2] / 64
+            coords[i, :, 0] *= image.shape[-1] / self.output_shape[0]
+            coords[i, :, 1] *= image.shape[-2] / self.output_shape[1]
```

Comment:
Have you double-checked that these are the correct way around? `output_shape` will be (height, width), but I can't recall whether `coords` is (x, y) or (y, x).